techniques/html/H101.html

```diff
@@ -97,29 +97,29 @@ <h2 class="box-h box-h-icon">About this Technique</h2>
     
     
     <section id="description">
-      <h2>Description</h2>
-      <p>The purpose of this technique is to provide programmatic access to sections of a web page. Landmark roles (or "landmarks") programmatically identify sections of a page. Browsers don't expose landmarks directly to users, but they they are made available to them by assistive technology and browser extensions to help navigation to various sections of a page.</p>
-      <p>Landmarks provide an easy way for users of assistive technology understand the programmatic structure of a page and skip over content they're not interested in. For instance, if a site's header, global navigation, main content area, and footer are marked up as landmarks, a screen reader user can easily skip to the landmark they're interested in without having to work their way through all the preceding content. A blind user who may be familiar with a news site's menu, and is only interested in getting to the top story, could easily navigate to the "main" landmark, and bypass dozens of menu links. In another circumstance, a user who is blind may want to quickly find a navigation menu, and can do so by jumping to the navigation landmark.</p>
-      <p>Landmark regions are implicitly created when certain semantic HTML elements are used, which is different from <a href="../aria/ARIA11">using ARIA to explicitly create landmarks</a>. These elements are:</p>
+      <h2>解説</h2>
+      <p>このテクニックの目的は、ウェブページのセクションへのプログラムによるアクセスを提供することである。ランドマークロール (又は「ランドマーク」) は、プログラムによってページのセクションを識別する。ブラウザはランドマークを利用者に直接公開しないが、ランドマークは支援技術及びブラウザ拡張機能によって、ページの様々なセクションへのナビゲーションを支援するために利用できる。</p>
+      <p>ランドマークは、支援技術を使用する利用者がページのプログラム構造を理解し、関心のないコンテンツをスキップする容易な方法を提供する。例えば、サイトのヘッダー、グローバルナビゲーション、メインコンテンツエリア、フッターがランドマークとしてマークアップされている場合、スクリーンリーダーを使用する利用者は、先行するコンテンツを全て確認することなく、関心のあるランドマークに容易に移動できる。ニュースサイトのメニューに精通していて、トップニュースにのみアクセスしたい視覚障害のある利用者は、"main" ランドマークに容易に移動でき、多数のメニューリンクをバイパスできる。別の状況では、視覚障害のある利用者がナビゲーションメニューを素早く見つけたい場合、ナビゲーションランドマークにジャンプすることで見つけることができる。</p>
```

Comment:
最後の文、「見つけたい場合……見つけることができる」は冗長なように思いました。can do so に対して補っていただいたのだと思いますが、原文そのまんま直訳の「それが可能になる」でも通じるのではないでしょうか。

```suggestion
      <p>ランドマークは、支援技術を使用する利用者がページのプログラム構造を理解し、関心のないコンテンツをスキップする容易な方法を提供する。例えば、サイトのヘッダー、グローバルナビゲーション、メインコンテンツエリア、フッターがランドマークとしてマークアップされている場合、スクリーンリーダーを使用する利用者は、先行するコンテンツを全て確認することなく、関心のあるランドマークに容易に移動できる。ニュースサイトのメニューに精通していて、トップニュースにのみアクセスしたい視覚障害のある利用者は、"main" ランドマークに容易に移動でき、多数のメニューリンクをバイパスできる。別の状況として、視覚障害のある利用者がナビゲーションメニューを素早く見つけたい場合、ナビゲーションランドマークにジャンプすることでそれが可能になる。</p>
```